packages/apps/redis/values.yaml

```diff
@@ -1,2 +1,3 @@
+replicas: 3
```

Comment:
Why three replicas? 2 is enough for redis master-slave

Reply:
fixed, thank you